src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/_bootstrap-variables.scss

```diff
@@ -416,97 +426,99 @@ $navbar-inverse-toggle-border-color:        #333 !default;
-$pagination-disabled-bg:               #fff !default;
-$pagination-disabled-border:           #ddd !default;
+$pagination-disabled-color:            $gray-light;
+$pagination-disabled-bg:               $navbar-default-bg;
+$pagination-disabled-border:           $gray-lighter;
 
 
 //== Pager
 //
 //##
 
-$pager-bg:                             $pagination-bg !default;
-$pager-border:                         $pagination-border !default;
-$pager-border-radius:                  15px !default;
+$pager-color:                          $pagination-color;
+$pager-bg:                             $pagination-bg;
+$pager-border:                         $pagination-border;
+$pager-border-radius:                  15px;
 
-$pager-hover-bg:                       $pagination-hover-bg !default;
+$pager-hover-color:                          $pagination-hover-color;
+$pager-hover-bg:                       $pagination-hover-bg;
 
-$pager-active-bg:                      $pagination-active-bg !default;
-$pager-active-color:                   $pagination-active-color !default;
+$pager-active-bg:                      $pagination-active-bg;
+$pager-active-color:                   $pagination-active-color;
 
-$pager-disabled-color:                 $pagination-disabled-color !default;
+$pager-disabled-color:                 $pagination-disabled-color;
 
 
 //== Jumbotron
 //
 //##
 
-$jumbotron-padding:              30px !default;
-$jumbotron-color:                inherit !default;
-$jumbotron-bg:                   $gray-lighter !default;
-$jumbotron-heading-color:        inherit !default;
-$jumbotron-font-size:            ceil(($font-size-base * 1.5)) !default;
+$jumbotron-padding:              30px;
+$jumbotron-color:                inherit;
+$jumbotron-bg:                   $gray-lightest;
+$jumbotron-heading-color:        inherit;
+$jumbotron-font-size:            ceil(($font-size-base * 1.5));
 
 
 //== Form states and alerts
 //
 //## Define colors for form feedback states and, by default, alerts.
 
-$state-success-text:             #fff;
-$state-success-bg:               #56b989;
-$state-success-border:           darken(adjust-hue($state-success-bg, -10), 5%) !default;
+$state-success-text:             $white;
+$state-success-bg:               $btn-success-bg;
+$state-success-border:           darken(adjust-hue($state-success-bg, -10), 5%);
 
-$state-info-text:                #fff;
-$state-info-bg:                  #b6bcd0;
-$state-info-border:              darken(adjust-hue($state-info-bg, -10), 7%) !default;
+$state-info-text:                $white;
+$state-info-bg:                  $brand-info;
+$state-info-border:              darken(adjust-hue($state-info-bg, -10), 7%);
 
-$state-warning-text:             #fff;
-$state-warning-bg:               #d0b15c;
-$state-warning-border:           darken(adjust-hue($state-warning-bg, -10), 5%) !default;
+$state-warning-text:             $white;
+$state-warning-bg:               #eec733;
```

Comment:
Deze ook in een variabele steken. Je mag gerust nieuwe variabelen definieren, zoals $brand-warning-light in dit geval.